src/chrome/content/rules/Wolfram_Alpha.xml

```diff
@@ -1,102 +1,47 @@
+	dev2.cvs.wolframalpha.com
+	reg.cvs.wolframalpha.com
+	sit.cvs.wolframalpha.com
+	cvsrx.wolframalpha.com
+	cvsrx.devel.wolframalpha.com
+	developer.wolframalpha.com
+	www-cn.wolframalpha.com
+	www-tw.wolframalpha.com
+
+Refused: 
+	community.wolframalpha.com
+    
+Timeout: 
+	blog.wolframalpha.com
+	nba.devel.wolframalpha.com
+	homeworkday.wolframalpha.com
+	m.wolframalpha.com
+	products.wolframalpha.com
+	www2.wolframalpha.com
 -->
-
-<!-- this ruleset creates mixed content, but it doesn't seem to break anything anymore -->
-<ruleset name="Wolfram Alpha.com (experimental)" default_off="Breaks login">
-
-	<target host="wolframalpha.com" />
-	<target host="*.wolframalpha.com" />
-		<!--
-			These break search:
-								-->
-		<exclusion pattern="^http://www\d\w?\.wolframalpha\.com/input/(?:pod|recalculate)\.jsp" />
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^(www|www\d\w)\.wolframalpha\.com$" name="^JSESSIONID$" /-->
-	<!--securecookie host="\.wolframalpha.com$" name="^(WR_SID|WolframAlphaNoProCallouts|WolframHomepageVisits|WolframResultsPageVisits|wa_pro_first|wa_pro_firstprovisit)$" /-->
-	<!--securecookie host="^www.wolframalpha.com$" name="^(CookieWarning|session_protocol|WolframAlphaChromeExtension)$" /-->
-	<!--securecookie host="^(.*\.)?wolframalpha\.com$" name=".+"/-->
-	<!--
-		Could we secure any of these safely?
-							-->
-	<!--securecookie host="^\.wolframalpha\.com$" name="^(wa_pro_first|wa_pro_firstprovisit|WolframAlphaSessionVisits|WolframResultsPageVisits|WR_SID)$" /-->
-	<!--
-		Tracking cookies:
-					-->
-	<securecookie host="^\.wolframalpha\.com$" name="^__utm\w$" />
-	<securecookie host="^(?:volunteer|www)\.wolframalpha\.com$" name=".+" />
-
-
-	<rule from="^http://(?:www\.)?wolframalpha\.com/"
-		to="https://www.wolframalpha.com/" />
-
-	<rule from="^http://(volunteer|www\d\w?)\.wolframalpha\.com/"
-		to="https://$1.wolframalpha.com/" />
-
+<ruleset name="Wolfram Alpha">
+	<target host="www.wolframalpha.com" />
+	<target host="api.wolframalpha.com" />
+	<target host="api-cn.wolframalpha.com" />
+	<target host="api-maps.wolframalpha.com" />
+	<target host="api-tw.wolframalpha.com" />
+	<target host="preview.wolframalpha.com" />
+	<target host="volunteer.wolframalpha.com" />
+	<target host="wc.wolframalpha.com" />
+	<target host="www1.wolframalpha.com" />
+	<target host="www10.wolframalpha.com" />
+	<target host="www3.wolframalpha.com" />
+	<target host="www4b.wolframalpha.com" />
+	<target host="www4c.wolframalpha.com" />
+	<target host="www4d.wolframalpha.com" />
+	<target host="www4f.wolframalpha.com" />
+	<target host="www5a.wolframalpha.com" />
+	<target host="www5b.wolframalpha.com" />
+	
+	<rule from="^http://api-(cn|maps|tw)\.wolframalpha\.com/" to="https://www.wolframalpha.com/" />
+	<rule from="^http://volunteer\.wolframalpha\.com/" to="https://www.wolframalpha.com/" />
+	<rule from="^http://(www[0-9][0-9]\.)wolframalpha\.com/" to="https://www.wolframalpha.com/" />
```

Comment:
Please document why those redirects are needed.
It seems that any `www\d\d` subdomain works, so you can use a wildcard target here.